src/Homework1.java

```diff
@@ -1,101 +1,224 @@
+            System.out.println("TRUE");
+        } else {
+            same = false;
+            System.out.println("FALSE");
+        }
+
+        //5. Write a program that test whether 2 int arrays are equal. Equality in this case means that both arrays have the same number of elements an element on a specific index in first array is equal to the element on the same index in the second array.
+        int first[] = {1, 2, 3, 4};
+        int second[] = {1, 2, 3};
+        boolean same2 = false;
+        if (first.length == second.length) {
+            same2 = true;
+            System.out.println("TRUE");
+        } else if (first.length != second.length) {
+            same2 = false;
+            System.out.println("FALSE");
+        }
+
+
+        //6. Write a java program that will concatenate 2 arrays.
+        int myFirstAr[] = {1, 2, 3, 4};
+        int mySecondAr[] = {5, 6, 7, 8};
+        int concatenateAr[];
+        concatenateAr = new int[myFirstAr.length + mySecondAr.length];
+        for (int i = 0; i < myFirstAr.length; i++) {
+            concatenateAr[i] = myFirstAr[i];
+        }
+        for (int i = 0; i < mySecondAr.length; i++) {
+            concatenateAr[myFirstAr.length + i] = mySecondAr[i];
+        }
+        for (int i = 0; i < concatenateAr.length; i++) {
+            System.out.println(concatenateAr[i]);
+        }
 
-        float myFloatPrimitive = 10.5f;
-        double myDoublePrimitive = 10.5f;
-        System.out.println(myFloatPrimitive);
 
-        char myCharPrimitive = 'c';
+        //7. Write a java program that will return the min, max and average values in a given array.
+        int givenArr[] = {13, 20, 14, 5, 2, 8};
+        int maxValue = givenArr[0];
+        int minValue = givenArr[0];
+        //max
+        for (int i = 0; i <= givenArr.length - 1; i++) {
+            if (givenArr[i] > maxValue) {
+                maxValue = givenArr[i];
+                System.out.println("The MAX value is " + maxValue);
+            }
+        }
+        //min
+        for (int i = 0; i <= givenArr.length - 1; i++) {
+            if (givenArr[i] < minValue) {
+                minValue = givenArr[i];
+                System.out.println("The MIN value is " + minValue);
+            }
+        }
+        //average
+        int sumGivenAr = 0;
+        for (int j = 0; j <= givenArr.length - 1; j++) {
+            sumGivenAr = sumGivenAr + givenArr[j];
+        }
+        int averageGivenAr = sumGivenAr / givenArr.length - 1;
+        System.out.println("The average of the given array is " + averageGivenAr);
 
-        boolean myBooleanPrimitive = true;
-        boolean myFalseBooleanPrimitive = false;
 
-        // Objects
-        Integer myIntegerPrimitive = 10;
+        //8. Write a java program to return the index of a target element.
+        int targetElem[] = {13, 20, 14, 5, 2, 8};
+        for (int i = 0; i <= targetElem.length - 1; i++) {
+            targetElem[i] = targetElem[i] - 1;
```

Comment:
Nu asta era cerinta, de exemplu pentru sirul tau trebuia sa returnezi indexul lui 5 de exemplu, adica 3 ( 5 se afla pe pozitia 3 in sirul tau)